package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "desktop-hotkeys",
-  "version": "1.0.3",
+  "version": "1.4.3",
```

Comment:
Any reason why the version jumped from 1.0 to 1.4 and not 1.1?

Reply:
There were several (published) releases in between